geopandas/tests/test_geoseries.py

```diff
@@ -158,8 +158,7 @@ def test_transform(self):
     def test_fillna(self):
         # default is to fill with empty geometry
         na = self.na_none.fillna()
-        assert isinstance(na[2], BaseGeometry)
-        assert na[2].is_empty
+        assert not na[2]
```

Comment:
Why was this changed? Isn't it the default of fillna to fill with empty geometry? so shouldn't assert na[2].is_empty still work?

Reply:
Reverted.  I think that this test was changed for a previous commit in this PR that has since been overwritten.